corehq/apps/userreports/management/commands/count_ucrs.py

```diff
@@ -0,0 +1,38 @@
+from __future__ import absolute_import
+from __future__ import print_function
+
+import textwrap
+from datetime import datetime
+
+from django.conf import settings
+from django.core.management import BaseCommand
+
+from dimagi.utils.couch.database import iter_docs
+from corehq.dbaccessors.couchapps.all_docs import get_doc_ids_by_class
+
+from corehq.apps.userreports.models import ReportConfiguration, StaticReportConfiguration
+
+
+class Command(BaseCommand):
+    help = "Pull stats about UCR and report-builder reports server-wide"
+
+    def handle(self, **options):
+        config_ids = get_doc_ids_by_class(ReportConfiguration)
+
+        builder_count, ucr_count = 0, 0
+        for doc in iter_docs(ReportConfiguration.get_db(), config_ids):
+            if doc['report_meta']['created_by_builder']:
```

Comment:
There's also a `edited_manually` attribute in this meta report. Depending on who this data is for, it may be worth knowing the difference between reports started by builder and the ones that needed manual editing